thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -430,9 +441,6 @@ func (r *EvalFnRunner) validateOptionalFlags() error {
 		}
 	}
 
-	if err := cmdutil.ValidateImagePullPolicyValue(r.ImagePullPolicy); err != nil {
-		return err
-	}
```

Comment:
kinda nice that the validation logic is now encapsulated in the type and will be invoked while setting the value.